.chloggen/datadogexporter-enable-retry-in-host-metric-export.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enable retries in exportHostMetrics in Datadog traces_exporter
```

Comment:
Please see this comment: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18271/files#r1097082925

Reply:
I would not use 'internal host metrics', maybe 'usage metrics' is a better term? These metrics are used by end-users to understand where they have Collectors running, they are documented in-app (description [here](https://github.com/DataDog/integrations-core/blob/8c8ad8ff46997288c52c644766cac7282ec872b5/otel/metadata.csv#L30)), so I would err on the side of documenting this on the changelog.

Reply:
Usage metrics sounds better to me, updated changelog and the func name.

The error when exporting these metrics is visible to users, enabling retries will help reduce such error logs, so I think it is worth adding a changelog.